### ISSUE TITLE
Accessibility issues on show more pattern for provider selection

### ISSUE THREAD

**Issue by jbalboni** [department-of-veterans-affairs/va.gov-team#17317]
Our show more link on the provider selection page should follow the guidelines listed here:

https://dsva.slack.com/archives/C8E985R32/p1601055663006900

- insert the next set of items after the current set and before the “Load more” button
- move focus to the first item in the new set of items
- The button text should be as explicit as possible “load more ‘health and safety’ articles.” Some of this text could be wrapped in spans with .sr-onlys to visually hide them, e.g. if we just want the button to visually show “load more”.
- Make sure there’s a bit of text describing the list, such as “Showing 20 results for ‘health and safety’ articles”. It can be visually hidden, if required.
- add aria-live="polite" role="status" aria-atomic="true" to it.
- update the text after new items have been added to the list.

We may need to make some of these adjustments on the facility page, too (aside from the aria-live region).
